pingdom/resource_pingdom_check.go

```diff
@@ -243,6 +373,34 @@ func resourcePingdomCheckRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("hostname", ck.Hostname)
 	d.Set("name", ck.Name)
 	d.Set("resolution", ck.Resolution)
+	d.Set("sendtoandroid", ck.SendToAndroid)
+	d.Set("sendtoemail", ck.SendToEmail)
+	d.Set("sendtoiphone", ck.SendToIPhone)
+	d.Set("sendtosms", ck.SendToSms)
+	d.Set("sendtotwitter", ck.SendToTwitter)
+	d.Set("sendnotificationwhendown", ck.SendNotificationWhenDown)
+	d.Set("notifyagainevery", ck.NotifyAgainEvery)
+	d.Set("notifywhenbackup", ck.NotifyWhenBackup)
+	d.Set("hostname", ck.Hostname)
```

Comment:
[minor] looks redundant to set `hostname` twice but I can fix post-merge since it's just in the test